packages/conduit-rxjs-react/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.7.0
+
+Published TBD
+
+1. Updated dependencies to latest version.
+
+2. Updated React lifecycle method name for use with React 16.3 to 17.x. This is a temporary fix as opposed to a more permanent fix discussed in [Issue #3](https://github.com/indiana-university/conduit/issues/3).
+
+3. Switched `rxjs/operators` imports to `rxjs` since they moved to top level in RxJS v7.2.
```

Comment:
This could be updated with a direct link.

```
3. Switched `rxjs/operators` imports to `rxjs` since they moved to top level in [RxJS v7.2](https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#720-2021-07-05).
```